source/ccci-carleton-college/v1/menu.js

```diff
@@ -1,7 +1,8 @@
-import {get, ONE_DAY, ONE_HOUR} from '@frogpond/ccc-lib'
```

Comment:
Did this file get renamed from Olaf to Carleton? (GitHub Mobile throws an error when I try to comment on the file, ignore the line)

Reply:
No, but I think the contents are identical (or almost identical)